discover/deployment/articles-dxp/01-installing-search-engine/07-xpack-monitoring.markdown

```diff
@@ -0,0 +1,197 @@
+    settings.
+
+3.  Download and install the 
+[Liferay Connector to X-Pack Monitoring](https://www.liferay.com/marketplace)
+[Elastic Stack 6.x].
+
+4.  Configure the connector to communicate with Elasticsearch.
+
+This document assumes you're enabling security *and* monitoring, though it's not
+a requirement. Differences in the process are noted as appropriate.
+
+For the X-Pack installation procedure, refer to the 
+[X-Pack security article](/discover/deployment/-/knowledge_base-7-0/securing-elasticsearch-6-with-x-pack).
+
+This guide starts with the installation of Kibana.
+
+## Install Kibana [](id=install-kibana)
+
+Make sure to install the correct version of Kibana. Check the 
+[Liferay Enterprise Search compatibility matrix](https://web.liferay.com/group/customer/dxp/support/compatibility-matrix/enterprise-search)
+for details.
+
+1.  [Download Kibana](https://www.elastic.co/downloads/kibana) and extract it.
+    The root folder is referred to as *Kibana Home*.
+
+2.  Install X-Pack into Kibana:
+
+        ./bin/kibana-plugin install x-pack
+
+    If you're not using X-Pack Security on the Elasticsearch cluster, skip this
+    step.
+
+3.  Tell Kibana where to send monitoring data by setting Elasticsearch's URL in
+    `kibana.yml`:
+
+        elasticsearch.url: "http://localhost:9200"
+
+    If SSL is enabled on Elasticsearch, this is an `https` URL.
+
+4. If not using X-Pack security, start Kibana by entering
+
+        ./bin/kibana
+
+    from Kibana Home.
+
+If you're using X-Pack's security features, there's additional configuration
+required before starting Kibana.
+
+### Configure Kibana with Authentication [](id=configure-kibana-with-authentication)
+
+If X-Pack requires authentication to access the Elasticsearch cluster, follow
+these steps or refer to 
+[Elastic's documentation](https://www.elastic.co/guide/en/kibana/6.1/monitoring-xpack-kibana.html). 
+
+1.  Set the password for the built-in `kibana` user in `Kibana
+    Home/config/kibana.yml`:
+
+        elasticsearch.username: "kibana"
+        elasticsearch.password: "liferay"
+
+    The password is whatever you set it to when initially setting up X-Pack.
+    Once Kibana is installed, you can change the built-in user passwords from the
+    *Management* user interface.
+
+2.  If you're not encrypting communication with the Elasticsearch cluster, start
+    Kibana with 
+
+        ./bin/kibana
+
+    and navigate to `localhost:5601`. Log in with a 
+    [user](https://www.elastic.co/guide/en/x-pack/6.1/native-realm.html#native-add)
+    who has the `kibana_user` 
+    [role](https://www.elastic.co/guide/en/x-pack/6.1/built-in-roles.html).
+
+### Configuring Kibana with Encryption [](id=configuring-kibana-with-encryption)
+
+Follow these steps to configure Kibana if X-Pack encrypts communication with the
+Elasticsearch cluster. Consult 
+[Elastic's guide](https://www.elastic.co/guide/en/kibana/6.2/using-kibana-with-security.html#using-kibana-with-security)
```

Comment:
We should switch this link to the one for 6.1
https://www.elastic.co/guide/en/kibana/6.1/using-kibana-with-security.html#using-kibana-with-security

Reply:
Nice catch--thanks!